### DISCUSSION TITLE
Admin panel does not remove users

### DISCUSSION THREAD

**Discussion by iBobik** [contember/admin#557] [Support]
When admin use Revoke user in the users list it will remove it from the project, but user can still sign in, but content requests fails on 404.

I can handle it by a message like "Account is not active" but it would be nice to let admins delete account by default when user is only in the single project.

**Comment by honzasladek** [comment]
You are correct. First step is we'll allow to disable the user completely (already PR here: https://github.com/contember/engine/pull/440), next step (ASAP) is also to allow delete the user completely.
